xml/Currency.xml

```diff
@@ -1,7 +1,7 @@
 <codelist name="Currency" xml:lang="en" complete="1">
     <metadata>
         <name>
-            <narrative>Currency</narrative>
+            <narrative>&gt;Currency</narrative>
```

Comment:
Yep, thanks. Fixed in https://github.com/IATI/IATI-Codelists-NonEmbedded/commit/a5ec422bd3344d9d733902406520715b34fb7e92